### PR TITLE
Fixes Issue #1 Added Resist Physical and Resist Poison Potion.

### PR DESCRIPTION
**Pull request by majordwarf**
Added 2 new potion prefabs. Didn't included the speed potion prefab as it already existed under name "RecipeWalkSpeedPotion".

### CODE REVIEW

**Comment by syntaxi** [comment]
You seem to have something mixed up with the code? The recipe for the *ResistPhysicalPotion* and actually contains the speed potion recipe. And the recipe for *ResistPoisonPotion* is actually for the resist physical potion

**Comment by majordwarf** [comment]
I see, some of the changes made in IntelliJ were not saved and I committed using GitKraken so I forgot to cross check.

**Comment by Cervator** [comment]
I can try to check on this a bit more but two questions first @majordwarf :-)

* Could you check on your Git config locally on whatever system you worked on for this? Your commits aren't connecting to your GitHub user. Usually that's from not having an associated email in both Git and GitHub.
* Have you been able to test these recipes in-game? Do they work? Could you supply easy copy-paste-able commands needed to fetch the crafting station and ingredients? That'd make it super easy to test 👍

File: assets/prefabs/recipe/brew/RecipeResistPhysicalPotion.prefab [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+            "Potions:RegenPotion": 1,
+            "Alchemy:HealingHerb": 2,
+        }
+    },
+    "InventoryOutput": {
+        "itemCounts": {
+            "Potions:ResistPhysicalPotion": 1
+        }
+    }
+}
```
Comment (e-aakash) [inline assets/prefabs/recipe/brew/RecipeResistPhysicalPotion.prefab:33]: It is good practice to add newline at end of file.
same for other file added

File: assets/prefabs/recipe/brew/RecipeResistPhysicalPotion.prefab [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        "recipeId": "Potions:ResistPhysicalPotion"
+    },
+    "ProcessingTime": {
+        "duration": 10000
+    },
+    "InventoryInput": {
+        "itemCounts": {
+            "Potions:ToughGlassBottle": 1,
+            "Potions:RegenPotion": 1,
+            "Alchemy:HealingHerb": 2,
```
Comment (syntaxi) [inline assets/prefabs/recipe/brew/RecipeResistPhysicalPotion.prefab]: You've got an extra comma here.
That'll cause it to spit out errors

File: assets/prefabs/recipe/brew/RecipeResistPoisonPotion.prefab [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        "recipeId": "Potions:ResistPhysicalPotion"
+    },
+    "ProcessingTime": {
+        "duration": 10000
+    },
+    "InventoryInput": {
+        "itemCounts": {
+            "Potions:ToughGlassBottle": 1,
+            "Potions:RegenPotion": 1,
+            "Alchemy:PoisonHerb": 2,
```
Comment (syntaxi) [inline assets/prefabs/recipe/brew/RecipeResistPoisonPotion.prefab]: Again, extra comma

**Review by syntaxi — CHANGES_REQUESTED** [review]

**Pull request merged**